src/main/java/dev/jbang/util/VersionChecker.java

```diff
@@ -0,0 +1,124 @@
+package dev.jbang.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.FileTime;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+
+import dev.jbang.Settings;
+
+public class VersionChecker {
+	private static final String jbangVersionUrl = "https://github.com/jbangdev/jbang/releases/latest/download/version.txt";
```

Comment:
Please use `https://www.jbang.dev/releases/latest/download/version.txt` with follow redirect enabled.

then we get a poor-mans stat on checks.

Reply:
ok, I didn´t know about that one. Shouldn't we then do that for downloads as well?

Reply:
I just created. I have 3 free rules. This is the last rule I have left :)

Reply:
Not sure I understand. You have 3 free rules. On is for version.txt. BUt what are the other 2 then?

Reply:
Sh1.Jbang and ps1.jBang

Reply:
Ah yes. Hmmm. Does GitHub provide download number at least for the releases?

Reply:
Yes. There are download stats per artifact.

Reply:
just realized I could use `*` and `$1` to have it work for multiple files - so now `www.jbang.dev/releases/latest/download/jbang.zip` works too.

Reply:
the benefit for these are that at least in theory we should be able to get the agent string on requests and get a bit more interesting data.